projects/affordance/utils.py

```diff
@@ -10,11 +10,24 @@
 from nltk.wsd import lesk
 from nltk.corpus import wordnet as wn
 from PyDictionary import PyDictionary
-from research.knowledge_base import KnowledgeFile, URI
+from knowledge_base import KnowledgeFile, URI
```

Comment:
Can you double check that this works? I'm surprised that we can import `knowledge_base` directly.

Reply:
It works and I am surprised as well. I though it's a Python3 new import change, but I what I found was is that Python3 require more explicit statement. https://stackoverflow.com/questions/12172791/changes-in-import-statement-python3 

Would you advise me to change to the above format? Or as long as it works, let's ignore it for now?